db/1.10/modules/ROOT/pages/database-documentation.adoc

```diff
@@ -0,0 +1,1165 @@
+
+
+==== Parameters
+[%header,cols="20s,20a,35a,20a,5a"]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use | | x
+| Transactional Action a| Enumeration, one of:
+
+** ALWAYS_JOIN
+** JOIN_IF_POSSIBLE
+** NOT_SUPPORTED |  The type of joining action that operations can take regarding transactions |  `JOIN_IF_POSSIBLE` |
+| Streaming Strategy a| * <<repeatable-in-memory-iterable>>
+* <<repeatable-file-store-iterable>>
+* non-repeatable-iterable |  Configure to use repeatable streams |  |
+| Query Timeout a| Number |  Indicates the minimum amount of time before the JDBC driver attempts to cancel a running statement. By default, no timeout is used. |  `0` |
+| Query Timeout Unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the `#queryTimeout` |  `SECONDS` |
+| Fetch Size a| Number |  Indicates how many rows to fetch from the database when rows are read from a ResultSet. This property is required when streaming is set to `true`, in which case a default value of `10` is used. |  |
+| Max Rows a| Number |  Sets the limit for the maximum number of rows that any ResultSet object generated by this message processor can contain for the given number. If the limit is exceeded, the excess rows are silently dropped. |  |
+| SQL Query Text a| String |  The text of the SQL query to execute |  | x
+| Parameter Types a| Array of <<ParameterType>> |  Allows you to optionally specify the type of one or more of the parameters in the query. If provided, you're not required to reference all of the parameters; but you cannot reference a parameter that is not present in the input values. |  |
+| Input Parameters a| Object |  A map in which keys are the name of an input parameter to be set on the JDBC prepared statement. Each parameter should be referenced in the SQL text using a colon prefix (for example: where id = :myParamName)).  The map's values will contain the actual assignation for each parameter. |  |
+| Target Variable a| String |  The name of a variable to store the operation's output. |  |
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+==== Output
+[%autowidth.spread]
+|===
+|Type |Array of Object
+|===
+
+=== For Configurations
+* <<config>>
+
+=== Working with Pooling Profiles
+When working with pooling profiles and the *Select* operation, the connection remains open until the flow execution ends or the content of the streams are consumed completely, or if the connection is the transaction key. This behavior occurs because the resultset the operation generates can have a stream or be part of an ongoing transaction.
```

Comment:
```suggestion
When working with pooling profiles and the *Select* operation, the connection remains open until one of the following occurs:

* The flow execution ends 
* The content of the streams are consumed completely
* The connection is the transaction key. 

This behavior occurs because the result set the operation generates can have a stream or be part of an ongoing transaction.
```